docs/public/debugging-using-devfile.adoc

```diff
@@ -52,7 +52,7 @@ commands:
 - Next we port forward a local port for debugging using `odo debug port-forward`. The default local port used for debugging is 5858. If 5858 is occupied, odo will automatically pick up a local port. We can also specify the local port using, `odo debug port-forward --local-port 5858`
 - Next we need to attach the debugger to the local port. Here's a guide to do it for VS Code : https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_remote-debugging[Remote Debugging]
 
-### Check if a debugging session is running
+## Check if a debugging session is running
```

Comment:
@prietyc123 Can you please more details why we need this ?

Reply:
It is pointing to some warning error like 

![image](https://user-images.githubusercontent.com/51620925/87511267-a987de00-c692-11ea-8610-7d96a0fb6862.png)

and this is just because after level0 we have directly used level2 which I have also mentioned in description.